fintual-oss/paraffin-back
# Feat learning units successions endpoint

# Context
- With this PR, we are creating and endpoint that lets the frontend ask for all the relations between learning units within a cycle.

# Changelog
- Create a new controller `learning_unit_successions_controller` to manage the data request.
- Create the route for the endpoint nested in `cycles/{cycle_id}`
- Create the spec for the endpoint
- Create swagger documentation for endpoint
- Slight refactor, deleting a comment found in the resources controller

# QA
- Go to [api-docs page](http://localhost:3001/api-docs/index.html)
- Test the endpoint. In the seeds there are already cycles with id's (1..3) and the first one already has some succession relations created. You have to run `rails db:seed` in your rails server in order to create the new seeds in your database.
- Check that it returns what is defined in the api-docs in each case (existing cycle, not authorized, non-existing cycle).
